### PR TITLE
Open licences on new tab

### LINKED ISSUE #412: Link to CC.org should open in a new tab
Link in "Learn more about CC licenses here." on the homepage should open in a new tab/window so that the user doesn't have to navigate back to CC Search after learning about the licenses.

### PR DESCRIPTION
**Pull request by geekayush**
Fixes #412 
The link "Learn more about CC licenses _here_" now opens in a separate tab.

### CODE REVIEW

File: src/components/HeroSection.vue [hunk 1]
```diff
@@ -26,7 +26,7 @@
         <p>
           Search for free content in the public domain and under Creative Commons licenses.
           <br />
-          Learn more about CC licenses <a href="https://creativecommons.org/share-your-work/licensing-types-examples/">here</a>.
+          Learn more about CC licenses <a href="https://creativecommons.org/share-your-work/licensing-types-examples/" target="_blank">here</a>.
```
Comment (brenoferreira) [inline src/components/HeroSection.vue]: Can you please add the `rel="noopener"` attribute in addition to `target="_blank"`?

For reference: https://developers.google.com/web/tools/lighthouse/audits/noopener

Comment (geekayush) [reply src/components/HeroSection.vue]: I didn't know about the 'rel' attribute. Thanks!

File: src/components/HeroSection.vue [hunk 2]
```diff
@@ -1,49 +1,52 @@
...
-          Learn more about CC licenses <a href="https://creativecommons.org/share-your-work/licensing-types-examples/">here</a>.
-        </p>
+    <div class='description'>
+      <p>
+        Search for free content in the public domain and under Creative Commons licenses.
+        <br />Learn more about CC licenses
+        <a
+          href='https://creativecommons.org/share-your-work/licensing-types-examples/'
+          target='_blank'
+          rel='noopener'
```
Comment (brenoferreira) [inline src/components/HeroSection.vue]: Thank you for adding this 👍

**Comment by geekayush** [comment]
> There are a lot of changes to the current code style that is followed in the rest of the codebase. Could you keep your changes limited to only the license link please?

I ran ```npm run lint``` and it gave no errors after which I pushed the changes. Why did that happen?  It should have produced errors or warnings. :confused:
I will revert back the linting issues though.

**Comment by kgodey** [comment]
@geekayush I have learned the hard way to always do a `git diff` before committing anything :)

**Comment by geekayush** [comment]
Never thought that even a little fix such as adding a target attribute to an anchor tag would let me learn two new things. :smile: 
Thanks! @kgodey @brenoferreira

**Comment by brenoferreira** [comment]
@geekayush `npm run lint` shouldn't write any changes to the codebase. Do you think maybe your text editor was configured to run some code formatter or something like that?

**Comment by geekayush** [comment]
@brenoferreira Yes, my code editor is configured. I beautified the code at my end.
Example: 
1. The strings had single quotes at the beginning.
```'string'```
2. It did the changes and saved the file. ( I have configured the editor to auto-beautify on save ). Now, according to my configuration, it changed the single-quotes into double-quotes.
```"string"```
3. After which I ran the lint command. Running ```npm run lint``` should have pointed out the single-quoted strings to be double-quoted.

**Pull request merged**